client/client_test.go

```diff
@@ -1653,6 +1654,48 @@ func testTarExporterWithSocket(t *testing.T, sb integration.Sandbox) {
 	require.NoError(t, err)
 }
 
+func testTarExporterWithSocketCopy(t *testing.T, sb integration.Sandbox) {
+	if os.Getenv("TEST_DOCKERD") == "1" {
+		t.Skip("tar exporter is temporarily broken on dockerd")
+	}
+
+	requiresLinux(t)
+	c, err := New(context.TODO(), sb.Address())
+	require.NoError(t, err)
+	defer c.Close()
+
+	alpine := llb.Image("docker.io/library/alpine:latest")
+	state := alpine.Run(llb.Args([]string{"sh", "-c", "nc -l -s local:/root/socket.sock & usleep 100000; kill %1"})).Root()
+
+	fa := llb.Copy(state, "/root", "/roo2",
+		&llb.CopyInfo{
+			FollowSymlinks:      true,
```

Comment:
Don't think these options are needed for the test

Reply:
i'll remove them; they had no impact on the test.